airflow/config_templates/default_celery.py

```diff
@@ -36,6 +36,12 @@ def _broker_supports_visibility_timeout(url):
     if _broker_supports_visibility_timeout(broker_url):
         broker_transport_options['visibility_timeout'] = 21600
 
+if conf.has_option("celery", 'RESULT_BACKEND'):
+    result_backend = str(conf.get('celery', 'RESULT_BACKEND'))
+else:
+    log.info("Value for celery result_backend not found. Using sql_alchemy_conn with db+ prefix.")
+    result_backend = f'db+{conf.get("database", "SQL_ALCHEMY_CONN")}'
```

Comment:
Let’s put this into a function. Logging on import time can be problematic due to startup order.